.github/workflows/tests.yml

```diff
@@ -19,6 +19,7 @@ jobs:
           bazelisk build --local_test_jobs=1 -- //... -//tests/ios/frameworks/mixed-source/sources-with-prebuilt-binaries/...
```

Comment:
This path `tests/ios/frameworks/mixed-source/sources-with-prebuilt-binaries/` needs to be updated